### ISSUE TITLE
Crash after DJISDKManager.getInstance().registerApp on an armeabi-v7a device with API level 27  (works on arm64-v8a)

### ISSUE THREAD

**Issue by dorovl** [dji-sdk/Mobile-SDK-Android#710]
**Important notes:**

- The problematic device is an armeabi-v7a Samsung Galaxy Tab A 10.1 from 2016 (SM-T580) with API level 27
- The device where is working is a Samsung Galaxy S7 edge (SM-G935F) with API level 28 (works probably also on the DJI Smart Controller)
- The public void onRegister(DJIError error) method is never reached
- Works with dji-sdk 4.13.1 **but not on 4.14-trial1**

**To reproduce the error:**

Clone the Android-ActivationAndBindingDemo from https://github.com/DJI-Mobile-SDK-Tutorials/Android-ActivationAndBindingDemo

Enter the com.dji.sdk.API_KEY in AnddroidManifest.xml

Change the minSdkVersion from 16 to 19 in app/build.gradle.

In app/build.gradle replace:

```
'com.dji:dji-sdk:4.11' with 'com.dji:dji-sdk:4.14-trial1'
'com.dji:dji-sdk-provided:4.11' with 'com.dji:dji-sdk-provided:4.14-trial1'
```

In app/build.gradle comment the following line:

`//apply from : "https://terra-1-g.djicdn.com/71a7d383e71a4fb8887a310eb746b47f/msdk/Android-CommonConfig/config_sample_all.gradle"`

Note: in the context of [rosetadrone ](https://github.com/The1only/rosettadrone/tree/work) we needed to add the following line in app/build.gradle:

`doNotStrip "*/*/libDJISDKLOGJNI.so"`

Without this line, the following exception is thrown:

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: sq.rogue.rosettadrone, PID: 11323
    java.lang.UnsatisfiedLinkError: dlopen failed: "/data/app/sq.rogue.rosettadrone-8GSIZT3QK2_BbnDtmCQ3uA==/lib/arm64/libDJISDKLOGJNI.so" .dynamic section has invalid offset: 0x41120, expected to match PT_DYNAMIC offset: 0x0
        at java.lang.Runtime.loadLibrary0(Runtime.java:1016)
        at java.lang.System.loadLibrary(System.java:1669)
        at dji.log.DJISDKLogEngine.<clinit>(Unknown Source:6)
        at dji.log.DJISDKLogEngine.getInstance(Unknown Source:0)
        at dji.log.DJILogFileManager.init(Unknown Source:54)
        at dji.log.DJILogHelper.initManagers(Unknown Source:7)
        at dji.log.DJILogHelper.initLogHelper(Unknown Source:10)
        at dji.log.DJILogHelper.init(Unknown Source:47)
        at dji.log.DJILog.init(Unknown Source:9)
        at dji.sdk.sdkmanager.DJISDKManager.initLog(Unknown Source:49)
        at dji.sdk.sdkmanager.DJISDKManager.registerApp(Unknown Source:10)
        at sq.rogue.rosettadrone.DJISimulatorApplication.onCreate(DJISimulatorApplication.java:159)
        at sq.rogue.rosettadrone.RDApplication.startLoginApplication(RDApplication.java:38)
        at sq.rogue.rosettadrone.ConnectionActivity.checkAndRequestPermissions(ConnectionActivity.java:131)
        at sq.rogue.rosettadrone.ConnectionActivity.onCreate(ConnectionActivity.java:280)
        at android.app.Activity.performCreate(Activity.java:7327)
        at android.app.Activity.performCreate(Activity.java:7318)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3096)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3259)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1950)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7073)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

Run the ActivationDemo on the armeabi-v7a device:

```
12/19 10:43:50: Launching 'app' on Physical Device.
App restart successful without requiring a re-install.
$ adb shell am start -n "com.dji.activationDemo/com.dji.activationDemo.ConnectionActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -D
Waiting for application to come online: com.dji.activationDemo.test | com.dji.activationDemo
Waiting for application to come online: com.dji.activationDemo.test | com.dji.activationDemo
Connected to process 14732 on device 'samsung-sm_t580-5203f2b2ea3db347'.
Capturing and displaying logcat messages from application. This behavior can be disabled in the "Logcat output" section of the "Debugger" settings page.
W/ActivityThread: Application com.dji.activationDemo is waiting for the debugger on port 8100...
I/System.out: Sending WAIT chunk
Waiting for application to come online: com.dji.activationDemo.test | com.dji.activationDemo
Connecting to com.dji.activationDemo
Capturing and displaying logcat messages from application. This behavior can be disabled in the "Logcat output" section of the "Debugger" settings page.
W/ActivityThread: Application com.dji.activationDemo is waiting for the debugger on port 8100...
I/System.out: Sending WAIT chunk
I/zygote: Debugger is active
I/System.out: Debugger has connected
    waiting for debugger to settle...
Connected to the target VM, address: 'localhost:6664', transport: 'socket'
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: debugger has settled (1456)
D/MDO: MDO if sIsMdo=false, sIsCheckMdo=false, uid = 10215
D/Proxy: mdoP is null: 10215
I/zygote: The ClassLoaderContext is a special shared library.
I/zygote: Do partial code cache collection, code=18KB, data=19KB
    After code cache collection, code=17KB, data=19KB
    Increasing code cache capacity to 128KB
I/zygote: Do partial code cache collection, code=23KB, data=55KB
I/zygote: After code cache collection, code=23KB, data=55KB
I/zygote: Increasing code cache capacity to 256KB
D/SDKRelativeJNI: load lib suc
D/NetworkSecurityConfig: No Network Security Config specified, using platform default
E/com.dji.activationDemo.ConnectionActivity: onResume
D/OpenGLRenderer: HWUI GL Pipeline
D/InputTransport: Input channel constructed: fd=72
D/ViewRootImpl@e462ff2[ConnectionActivity]: setView = DecorView@2a154c0[ConnectionActivity] TM=true MM=false
D/InputTransport: Input channel constructed: fd=73
D/ViewRootImpl@8a6fcec[Toast]: setView = android.widget.LinearLayout{ab1c8b5 V.E...... ......I. 0,0-0,0} TM=true MM=false
V/Toast: Text: segi in android.widget.Toast$TN@9d7cb4a
D/ViewRootImpl@e462ff2[ConnectionActivity]: dispatchAttachedToWindow
V/Surface: sf_framedrop debug : 0x4f4c, game : false, logging : 0
D/ViewRootImpl@e462ff2[ConnectionActivity]: Relayout returned: old=[0,0][0,0] new=[0,0][1200,1920] result=0x7 surface={valid=true 3401820160} changed=true
I/zygote: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
I/OpenGLRenderer: Initialized EGL, version 1.4
D/OpenGLRenderer: Swap behavior 2
D/libGLESv1: STS_GLApi : DTS, ODTC are not allowed for Package : com.dji.activationDemo
D/mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, egl_color_buffer_format *, EGLBoolean) returns 0x3000,  [1200x1920]-format:1
D/OpenGLRenderer: eglCreateWindowSurface = 0xc7235248, 0xcac3a808
D/ViewRootImpl@8a6fcec[Toast]: dispatchAttachedToWindow
V/Surface: sf_framedrop debug : 0x4f4c, game : false, logging : 0
D/ViewRootImpl@8a6fcec[Toast]: Relayout returned: old=[0,0][0,0] new=[463,1758][737,1824] result=0x27 surface={valid=true 3401818112} changed=true
D/ViewRootImpl@e462ff2[ConnectionActivity]: MSG_RESIZED_REPORT: frame=Rect(0, 0 - 1200, 1920) ci=Rect(0, 36 - 0, 0) vi=Rect(0, 36 - 0, 0) or=1
D/vndksupport: Loading /vendor/lib/hw/android.hardware.graphics.mapper@2.0-impl.so from current namespace instead of sphal namespace.
D/ViewRootImpl@8a6fcec[Toast]: MSG_RESIZED_REPORT: frame=Rect(463, 1758 - 737, 1824) ci=Rect(0, 0 - 0, 0) vi=Rect(0, 0 - 0, 0) or=1
D/ViewRootImpl@e462ff2[ConnectionActivity]: MSG_WINDOW_FOCUS_CHANGED 1
V/InputMethodManager: Starting input: tba=android.view.inputmethod.EditorInfo@a12ae6c nm : com.dji.activationDemo ic=null
D/InputMethodManager: startInputInner - Id : 0
I/InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
D/InputTransport: Input channel constructed: fd=82
V/InputMethodManager: Starting input: tba=android.view.inputmethod.EditorInfo@aca98ca nm : com.dji.activationDemo ic=null
D/InputMethodManager: startInputInner - Id : 0
D/ViewRootImpl@8a6fcec[Toast]: dispatchDetachedFromWindow
D/InputEventReceiver: channel '93bbc0b Toast (client)' ~ Disposing input event receiver.
D/InputEventReceiver: channel '93bbc0b Toast (client)' ~NativeInputEventReceiver.
D/InputTransport: Input channel destroyed: fd=73
D/FPVController: try to load libdjivideo.so
D/NativeRcController: x try to load libusbdec.so
E/NativeRcController: Couldn't load libusbdec.so
I/onLoad: loaded onLoad.cpp :JNI_OnLoad() go to end
D/GroudStation: load lib suc
I/zygote: Do full code cache collection, code=100KB, data=124KB
    After code cache collection, code=95KB, data=64KB
D/DJIFlycParamInfoManager: readToMemory size = 705
E/DJIUsbAccessoryReceiver: start accessory receiver
D/UsbAccessoryService: accessory startStream
E/DJIUsbAccessoryReceiver: receive action: android.hardware.usb.action.USB_STATE
E/DJIUsbAccessoryReceiver: android.hardware.usb.action.USB_STATE: is connected
D/logWrite.cpp: UDT JNI_OnLoad---------------->
    UDP jniRegisterNativeMethods: jniRegisterNativeMethods
    loaded UDT :JNI_OnLoad() go to end
D/BluetoothAdapter: STATE_ON
I/chatty: uid=10215(com.dji.activationDemo) azonaws.com/... identical 1 line
D/BluetoothAdapter: STATE_ON
D/BluetoothAdapter: STATE_ON
D/BluetoothAdapter: STATE_ON
D/BluetoothAdapter: STATE_ON
I/chatty: uid=10215(com.dji.activationDemo) Timer-4 identical 1 line
D/BluetoothAdapter: STATE_ON
D/BluetoothLeScanner: Start Scan with callback
D/BluetoothLeScanner: onScannerRegistered() - status=0 scannerId=6 mScannerId=0
D/BluetoothAdapter: STATE_ON
D/BluetoothAdapter: STATE_ON
E/Lightbridge: Failed to obtain JNIEnv
E/Lightbridge: Failed to obtain JNIEnv
E/Lightbridge: Failed to obtain JNIEnv
E/Lightbridge: Failed to obtain JNIEnv
E/Lightbridge: Failed to obtain JNIEnv
E/Lightbridge: Failed to obtain JNIEnv
E/RM500 RCType: Service Not Connected
E/PM430 RCType: Service Not Connected
D/CertificatePolicyCache: Creating new instance of CertificatePolicyCache myUid: 10215 callingUid: 10215
    readVariables type: null userId: 0
I/System.out: Thread-1300(ApacheHTTPLog):Reading from variable values from setDefaultValuesToVariables
I/System.out: Thread-1300(ApacheHTTPLog):isSBSettingEnabled false
    Thread-1300(ApacheHTTPLog):isShipBuild true
I/System.out: Thread-1300(ApacheHTTPLog):getDebugLevel 0x4f4c
    Thread-1300(ApacheHTTPLog):Smart Bonding Setting is false
    Thread-1300(ApacheHTTPLog):SmartBonding Setting is false, SHIP_BUILD is true, log to file is false, DBG is false, DEBUG_LEVEL (1-LOW, 2-MID, 3-HIGH) is 1
D/HttpsHelper: verified, return
E/RM500 RCType: Service Not Connected
E/PM430 RCType: Service Not Connected
A/libc: Fatal signal 11 (SIGSEGV), code 2, fault addr 0xbf8f5000 in tid 14805 (azonaws.com/...), pid 14732 (.activationDemo)
Disconnected from the target VM, address: 'localhost:6664', transport: 'socket'
```

Logcat:
```
2020-12-19 10:42:45.035 14471-14471/com.dji.activationDemo E/Zygote: isWhitelistProcess - Process is Whitelisted
2020-12-19 10:42:45.036 14471-14471/com.dji.activationDemo E/libpersona: scanKnoxPersonas
2020-12-19 10:42:45.036 14471-14471/com.dji.activationDemo E/libpersona: Couldn't open the File - /data/system/users/0/personalist.xml - No such file or directory
2020-12-19 10:42:45.039 14471-14471/com.dji.activationDemo W/SELinux: SELinux selinux_android_compute_policy_index : Policy Index[2],  Con:u:r:zygote:s0 RAM:SEPF_SM-T580_8.1.0_0008, [-1 -1 -1 -1 0 1]
2020-12-19 10:42:45.040 14471-14471/com.dji.activationDemo I/SELinux: SELinux: seapp_context_lookup: seinfo=untrusted, level=s0:c512,c768, pkgname=com.dji.activationDemo 
2020-12-19 10:42:45.043 14471-14471/com.dji.activationDemo I/zygote: Late-enabling -Xcheck:jni
2020-12-19 10:42:45.152 14471-14471/com.dji.activationDemo D/TimaKeyStoreProvider: TimaKeyStore is not enabled: cannot add TimaSignature Service and generateKeyPair Service
2020-12-19 10:42:45.153 14471-14471/com.dji.activationDemo D/ActivityThread: Added TimaKeyStore provider
2020-12-19 10:42:45.251 14471-14471/com.dji.activationDemo W/ActivityThread: Application com.dji.activationDemo is waiting for the debugger on port 8100...
2020-12-19 10:42:45.252 14471-14471/com.dji.activationDemo I/System.out: Sending WAIT chunk
2020-12-19 10:42:46.041 14471-14478/com.dji.activationDemo I/zygote: Debugger is active
2020-12-19 10:42:46.054 14471-14471/com.dji.activationDemo I/System.out: Debugger has connected
2020-12-19 10:42:46.054 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:46.254 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:46.455 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:46.655 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:46.856 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:47.056 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:47.257 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:47.457 14471-14471/com.dji.activationDemo I/System.out: waiting for debugger to settle...
2020-12-19 10:42:47.658 14471-14471/com.dji.activationDemo I/System.out: debugger has settled (1439)
2020-12-19 10:42:47.662 14471-14471/com.dji.activationDemo D/MDO: MDO if sIsMdo=false, sIsCheckMdo=false, uid = 10215
2020-12-19 10:42:47.664 14471-14471/com.dji.activationDemo D/Proxy: mdoP is null: 10215
2020-12-19 10:42:47.670 14471-14471/com.dji.activationDemo I/zygote: The ClassLoaderContext is a special shared library.
2020-12-19 10:42:48.441 14471-14476/com.dji.activationDemo I/zygote: Do partial code cache collection, code=18KB, data=19KB
2020-12-19 10:42:48.441 14471-14476/com.dji.activationDemo I/zygote: After code cache collection, code=17KB, data=19KB
2020-12-19 10:42:48.441 14471-14476/com.dji.activationDemo I/zygote: Increasing code cache capacity to 128KB
2020-12-19 10:42:48.478 14471-14476/com.dji.activationDemo I/zygote: Do partial code cache collection, code=23KB, data=55KB
2020-12-19 10:42:48.478 14471-14476/com.dji.activationDemo I/zygote: After code cache collection, code=23KB, data=55KB
2020-12-19 10:42:48.479 14471-14476/com.dji.activationDemo I/zygote: Increasing code cache capacity to 256KB
2020-12-19 10:42:48.517 14471-14471/com.dji.activationDemo D/SDKRelativeJNI: load lib suc
2020-12-19 10:42:48.847 14471-14516/com.dji.activationDemo D/NetworkSecurityConfig: No Network Security Config specified, using platform default
2020-12-19 10:42:48.990 14471-14471/com.dji.activationDemo E/com.dji.activationDemo.ConnectionActivity: onResume
2020-12-19 10:42:49.006 14471-14522/com.dji.activationDemo D/OpenGLRenderer: HWUI GL Pipeline
2020-12-19 10:42:49.013 14471-14471/com.dji.activationDemo D/InputTransport: Input channel constructed: fd=73
2020-12-19 10:42:49.015 14471-14471/com.dji.activationDemo D/ViewRootImpl@f651f69[ConnectionActivity]: setView = DecorView@6c83223[ConnectionActivity] TM=true MM=false
2020-12-19 10:42:49.027 14471-14471/com.dji.activationDemo D/InputTransport: Input channel constructed: fd=74
2020-12-19 10:42:49.029 14471-14471/com.dji.activationDemo D/ViewRootImpl@161cd7f[Toast]: setView = android.widget.LinearLayout{27b614c V.E...... ......I. 0,0-0,0} TM=true MM=false
2020-12-19 10:42:49.030 14471-14471/com.dji.activationDemo V/Toast: Text: segi in android.widget.Toast$TN@2f5c595
2020-12-19 10:42:49.036 14471-14471/com.dji.activationDemo D/ViewRootImpl@f651f69[ConnectionActivity]: dispatchAttachedToWindow
2020-12-19 10:42:49.068 14471-14471/com.dji.activationDemo V/Surface: sf_framedrop debug : 0x4f4c, game : false, logging : 0
2020-12-19 10:42:49.069 14471-14471/com.dji.activationDemo D/ViewRootImpl@f651f69[ConnectionActivity]: Relayout returned: old=[0,0][0,0] new=[0,0][1200,1920] result=0x7 surface={valid=true 3403053056} changed=true
2020-12-19 10:42:49.078 14471-14522/com.dji.activationDemo I/zygote: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
2020-12-19 10:42:49.078 14471-14522/com.dji.activationDemo I/OpenGLRenderer: Initialized EGL, version 1.4
2020-12-19 10:42:49.078 14471-14522/com.dji.activationDemo D/OpenGLRenderer: Swap behavior 2
2020-12-19 10:42:49.084 14471-14522/com.dji.activationDemo D/libGLESv1: STS_GLApi : DTS, ODTC are not allowed for Package : com.dji.activationDemo
2020-12-19 10:42:49.085 14471-14522/com.dji.activationDemo D/mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, egl_color_buffer_format *, EGLBoolean) returns 0x3000,  [1200x1920]-format:1
2020-12-19 10:42:49.085 14471-14522/com.dji.activationDemo D/OpenGLRenderer: eglCreateWindowSurface = 0xdfe0eaa8, 0xcad67808
2020-12-19 10:42:49.092 14471-14471/com.dji.activationDemo D/ViewRootImpl@161cd7f[Toast]: dispatchAttachedToWindow
2020-12-19 10:42:49.115 14471-14471/com.dji.activationDemo V/Surface: sf_framedrop debug : 0x4f4c, game : false, logging : 0
2020-12-19 10:42:49.116 14471-14471/com.dji.activationDemo D/ViewRootImpl@161cd7f[Toast]: Relayout returned: old=[0,0][0,0] new=[463,1758][737,1824] result=0x27 surface={valid=true 3403051008} changed=true
2020-12-19 10:42:49.121 14471-14471/com.dji.activationDemo D/ViewRootImpl@f651f69[ConnectionActivity]: MSG_RESIZED_REPORT: frame=Rect(0, 0 - 1200, 1920) ci=Rect(0, 36 - 0, 0) vi=Rect(0, 36 - 0, 0) or=1
2020-12-19 10:42:49.122 14471-14471/com.dji.activationDemo D/ViewRootImpl@f651f69[ConnectionActivity]: MSG_WINDOW_FOCUS_CHANGED 1
2020-12-19 10:42:49.125 14471-14471/com.dji.activationDemo V/InputMethodManager: Starting input: tba=android.view.inputmethod.EditorInfo@f42d8ae nm : com.dji.activationDemo ic=null
2020-12-19 10:42:49.125 14471-14471/com.dji.activationDemo D/InputMethodManager: startInputInner - Id : 0
2020-12-19 10:42:49.125 14471-14471/com.dji.activationDemo I/InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
2020-12-19 10:42:49.133 14471-14484/com.dji.activationDemo D/InputTransport: Input channel constructed: fd=83
2020-12-19 10:42:49.152 14471-14522/com.dji.activationDemo D/vndksupport: Loading /vendor/lib/hw/android.hardware.graphics.mapper@2.0-impl.so from current namespace instead of sphal namespace.
2020-12-19 10:42:49.183 14471-14471/com.dji.activationDemo D/ViewRootImpl@161cd7f[Toast]: MSG_RESIZED_REPORT: frame=Rect(463, 1758 - 737, 1824) ci=Rect(0, 0 - 0, 0) vi=Rect(0, 0 - 0, 0) or=1
2020-12-19 10:42:49.184 14471-14471/com.dji.activationDemo V/InputMethodManager: Starting input: tba=android.view.inputmethod.EditorInfo@d420861 nm : com.dji.activationDemo ic=null
2020-12-19 10:42:49.184 14471-14471/com.dji.activationDemo D/InputMethodManager: startInputInner - Id : 0
2020-12-19 10:42:52.359 14471-14471/com.dji.activationDemo D/ViewRootImpl@161cd7f[Toast]: dispatchDetachedFromWindow
2020-12-19 10:42:52.360 14471-14471/com.dji.activationDemo D/InputEventReceiver: channel 'e151530 Toast (client)' ~ Disposing input event receiver.
2020-12-19 10:42:52.360 14471-14471/com.dji.activationDemo D/InputEventReceiver: channel 'e151530 Toast (client)' ~NativeInputEventReceiver.
2020-12-19 10:42:52.366 14471-14471/com.dji.activationDemo D/InputTransport: Input channel destroyed: fd=74
2020-12-19 10:42:55.772 14471-14544/com.dji.activationDemo D/FPVController: try to load libdjivideo.so
2020-12-19 10:42:55.982 14471-14577/com.dji.activationDemo D/NativeRcController: x try to load libusbdec.so
2020-12-19 10:42:56.060 14471-14551/com.dji.activationDemo I/onLoad: loaded onLoad.cpp :JNI_OnLoad() go to end
2020-12-19 10:42:56.060 14471-14551/com.dji.activationDemo D/GroudStation: load lib suc
2020-12-19 10:42:56.064 14471-14577/com.dji.activationDemo E/NativeRcController: Couldn't load libusbdec.so
2020-12-19 10:42:56.150 14471-14476/com.dji.activationDemo I/zygote: Do full code cache collection, code=100KB, data=124KB
2020-12-19 10:42:56.151 14471-14476/com.dji.activationDemo I/zygote: After code cache collection, code=95KB, data=64KB
2020-12-19 10:42:56.511 14471-14550/com.dji.activationDemo D/DJIFlycParamInfoManager: readToMemory size = 705
2020-12-19 10:42:56.609 14471-14544/com.dji.activationDemo E/DJIUsbAccessoryReceiver: start accessory receiver
2020-12-19 10:42:56.619 14471-14577/com.dji.activationDemo D/UsbAccessoryService: accessory startStream
2020-12-19 10:42:56.624 14471-14576/com.dji.activationDemo E/DJIUsbAccessoryReceiver: receive action: android.hardware.usb.action.USB_STATE
2020-12-19 10:42:56.625 14471-14576/com.dji.activationDemo E/DJIUsbAccessoryReceiver: android.hardware.usb.action.USB_STATE: is connected
2020-12-19 10:42:56.644 14471-14583/com.dji.activationDemo D/logWrite.cpp: UDT JNI_OnLoad---------------->
2020-12-19 10:42:56.644 14471-14583/com.dji.activationDemo D/logWrite.cpp: UDP jniRegisterNativeMethods: jniRegisterNativeMethods
2020-12-19 10:42:56.644 14471-14583/com.dji.activationDemo D/logWrite.cpp: loaded UDT :JNI_OnLoad() go to end
2020-12-19 10:42:56.688 14471-14544/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.688 14471-14544/com.dji.activationDemo I/chatty: uid=10215(com.dji.activationDemo) azonaws.com/... identical 1 line
2020-12-19 10:42:56.695 14471-14544/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.696 14471-14601/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.696 14471-14544/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.697 14471-14601/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.699 14471-14601/com.dji.activationDemo I/chatty: uid=10215(com.dji.activationDemo) Timer-4 identical 1 line
2020-12-19 10:42:56.700 14471-14601/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.702 14471-14601/com.dji.activationDemo D/BluetoothLeScanner: Start Scan with callback
2020-12-19 10:42:56.704 14471-14603/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.705 14471-14484/com.dji.activationDemo D/BluetoothLeScanner: onScannerRegistered() - status=0 scannerId=6 mScannerId=0
2020-12-19 10:42:56.706 14471-14603/com.dji.activationDemo D/BluetoothAdapter: STATE_ON
2020-12-19 10:42:56.741 14471-14605/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.741 14471-14606/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.742 14471-14608/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.742 14471-14610/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.744 14471-14607/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.751 14471-14609/com.dji.activationDemo E/Lightbridge: Failed to obtain JNIEnv
2020-12-19 10:42:56.882 14471-14544/com.dji.activationDemo E/RM500 RCType: Service Not Connected
2020-12-19 10:42:56.883 14471-14544/com.dji.activationDemo E/PM430 RCType: Service Not Connected
2020-12-19 10:42:57.318 14471-14544/com.dji.activationDemo D/CertificatePolicyCache: Creating new instance of CertificatePolicyCache myUid: 10215 callingUid: 10215
2020-12-19 10:42:57.318 14471-14544/com.dji.activationDemo D/CertificatePolicyCache: readVariables type: null userId: 0
2020-12-19 10:42:57.814 14471-14576/com.dji.activationDemo E/RM500 RCType: Service Not Connected
2020-12-19 10:42:57.815 14471-14576/com.dji.activationDemo E/PM430 RCType: Service Not Connected
2020-12-19 10:42:57.960 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):Reading from variable values from setDefaultValuesToVariables
2020-12-19 10:42:57.963 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):isSBSettingEnabled false
2020-12-19 10:42:57.963 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):isShipBuild true
2020-12-19 10:42:57.963 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):getDebugLevel 0x4f4c
2020-12-19 10:42:57.964 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):Smart Bonding Setting is false
2020-12-19 10:42:57.964 14471-14627/com.dji.activationDemo I/System.out: Thread-1295(ApacheHTTPLog):SmartBonding Setting is false, SHIP_BUILD is true, log to file is false, DBG is false, DEBUG_LEVEL (1-LOW, 2-MID, 3-HIGH) is 1
2020-12-19 10:42:58.259 14471-14627/com.dji.activationDemo D/HttpsHelper: verified, return
2020-12-19 10:42:58.320 14471-14544/com.dji.activationDemo A/libc: Fatal signal 11 (SIGSEGV), code 2, fault addr 0xbf9ad000 in tid 14544 (azonaws.com/...), pid 14471 (.activationDemo)
```

**Note**: "azonaws.com" showned as the tid 14544 is marked by Swisscom Mobile ISP as a "Dangerous website" which is blocked by default.

Note: to debug, one might change the debug build type to:
```
debug {
   //shrinkResources true
    //minifyEnabled true
    minifyEnabled false
    debuggable true
    jniDebuggable true
    proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
}
```

**Comment by Jai-GAY** [comment]
I got the following error on an Android 5.1.1 x86 device after this [new lib (libDJISDKLOGJNI.so) added](https://github.com/dji-sdk/Mobile-SDK-Android/blob/241af9a45b2753873c601b3988192bb202ed4c30/Sample%20Code/app/build.gradle#L63) in [4.14.1 release](https://github.com/dji-sdk/Mobile-SDK-Android/releases/tag/v4.14.1).

**DJISDKManager.getInstance().registerApp**
		at com.dji.sdk.sample.internal.view.MainContent$14.run(:646)
		03-24 15:57:47.797 5443-5472/com.wikitude.sdksamples E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
    Process: com.wikitude.sdksamples, PID: 5443
    java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[
	[zip file "/data/data/com.wikitude.sdksamples/.cache_sdk/sdkclasses2.jar",
	zip file "/data/data/com.wikitude.sdksamples/.cache_sdk/sdkclasses.jar", zip file
	"/data/app/com.wikitude.sdksamples-2/base.apk"],nativeLibraryDirectories=
	[/data/app/com.wikitude.sdksamples-2/lib/x86, /vendor/lib, /system/lib]]] couldn't find "libDJISDKLOGJNI.so"

**Comment by dji-dev** [comment]
Agent comment from William Wong in <a href='https://djisdksupport.zendesk.com/agent/tickets/42262'>Zendesk ticket #42262</a>:<br><br>Dear Client
Thank you for contacting DJI.

Unfortunately, we don't support X86 architecture anymore.

Hopefully our solution can help you.
Kindly Regards,
DJI Developer Support

**Comment by dji-dev** [comment]
Agent comment from Terje Nilsen in <a href='https://djisdksupport.zendesk.com/agent/tickets/42262'>Zendesk ticket #42262</a>:<br><br>Thank you for sorting that out, I will remove the option in the manifest and make a comment.

Best regard
Terje Nilsen
9Tek AS
Norway

On Mar 25, 2021, at 05:19, Support <dev@dji.com> wrote:

﻿

**Comment by dorovl** [comment]
> Agent comment from William Wong in [Zendesk ticket #42262](https://djisdksupport.zendesk.com/agent/tickets/42262):
> 
> Dear Client
> Thank you for contacting DJI.
> 
> Unfortunately, we don't support X86 architecture anymore.
> 
> Hopefully our solution can help you.
> Kindly Regards,
> DJI Developer Support

Hello @dji-dev,

Please confirm that you still support the armeabi-v7a architecture (32-bit ARM-based CPUs) which is still very spread.
Please also note that this issue is open since Dec 19, 2020.

Regards, Vlad

**Comment by DJI-William** [comment]
This issue is fixed in the latest 4.14.1 version, please tell us if the problem still exists

**Comment by DJI-William** [comment]
We still support armeabi-v7a architecture
